src/app/clusters/scenes-server/SceneHandlerImpl.cpp

```diff
@@ -20,6 +20,29 @@
 namespace chip {
 namespace scenes {
 
+namespace {
+
+/// ValideAttribute
+/// @brief  Validate the attribute exists for a given cluster
+/// @param[in] clusterID  Cluster ID
+/// @param[in] attID      Attribute ID
+/// @return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE if the attribute does not exist for a given cluster or is not scenable
+/// @note This will allways fail for global list attributes. If we do want to make them scenable someday, we will need to
```

Comment:
I don't see how those would ever be scenable, so that's probably not a concern. However, if the client passes in the attribute ID for a global attribute, will this function correctly fail?